### DISCUSSION TITLE
Is it possible to switch the sidebar according to the URL of the document?

### DISCUSSION THREAD

**Discussion by dninomiya** [facebook/docusaurus#7039] [Q&A] [answered]
For example, if I visit `tutorials/commonDoc` , I want `tutorialSidebar` to appear in the sidebar. I want you to see `apiSidebar` when you access `api/commonDoc`.

<img width="604" alt="image" src="https://user-images.githubusercontent.com/5842851/160366411-016082b5-2fee-4bba-bc07-71025e050df6.png">

**Comment by Josh-Cena** [comment, marked as answer]
Hi, sidebars are bound to docs, not to URLs. Here, `commonDoc` is a doc ID. And one doc can only appear at one route, so it's either `tutorial/commonDoc`, or `api/commonDoc`, but not both at the same time.

**Reply by dninomiya** [reply]
@Josh-Cena 
Thank you for your quick answer!
